meerk40t/core/elements.py

```diff
@@ -7391,36 +7610,73 @@ def classify(self, elements, operations=None, add_op_function=None):
             ######################
             # NON-CLASSIFIED ELEMENTS
             ######################
-            if not was_classified:
+            if not was_classified and usedefault:
+                # let's iterate through the default ops and add them
+                for op in operations:
+                    if hasattr(op, "classify"):
+                        classified, should_break = op.classify(node, fuzzy=fuzzy, fuzzydistance=fuzzydistance, usedefault=True)
+                    else:
+                        continue
+                    if classified:
+                        was_classified = True
+                    if should_break:
+                        break
+            if not was_classified and autogen:
+                # Despite all efforts we couldn't classify the element, so let's add an op
                 op = None
                 if node.type == "elem image":
                     op = ImageOpNode(output=False)
                 elif node.type == "elem point":
                     op = DotsOpNode(output=False)
-                elif hasattr(node, "stroke") and node.stroke is not None:
-                    if (
-                        node.stroke.red == 0xFF
-                        and node.stroke.blue == 0
-                        and node.stroke.green == 0
-                    ):
-                        op = CutOpNode(color=node.stroke, speed=5.0)
+                elif hasattr(node, "stroke") and node.stroke is not None and node.stroke.argb is not None:
+                    is_cut = Color.distance("red", node.stroke) <= 100
```

Comment:
I thought this would use `self.classify_fuzzydistance` rather than hardcode the distance.

Reply:
Amended